draft-ietf-rats-corim.md

```diff
@@ -2224,6 +2271,66 @@ The following transformation steps are applied for both the `identity-triples` a
 
 * If the Endorsement conceptual message has a profile, the profile is copied to the `ev`.`addition`.`profile` field.
 
+#### Domain Membership Triples Transformation {#sec-ir-dm-trans}
+
+This section describes how the external representation of a Domain Membership Triple (DMT) ({{sec-comid-triple-domain-membership}}) is transformed into its CoRIM internal representation ({{sec-ir-dm}}).
+
+{:dmt-enum: counter="dmt1" style="format Step %d."}
+
+{: dmt-enum}
+* Allocate a `dm` entry.
+
+* Set the conceptual message type for the domain-id ECT and each member ECT to 6 (`domain-member`).
```

Comment:
Not sure I follow: at step 1, we allocate the `dm` entry, which is supposedly empty (since we haven’t explained yet how to populate it).  Therefore, here we instruct on setting the `cm-type` field of ECTs that don’t exist?